src/EventPayload.ts

```diff
@@ -59,6 +60,24 @@ export interface EventPayload {
     /** The ID of the individual response in which the event occurred. */
     responseId?: string;
   };
+  /** Fields specific to reporting Search Analytics Events */
+  search?: {
+    /** Unique identifier of the search */
+    searchId: number,
+    /** Unique identifier for a single query across pagination */
+    queryId: number,
```

Comment:
I just realized while working on AE json schema change that searchId / queryId should be a string with 32 characters. regex like this: ^[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{12}$. Not sure if you need to do any validations here.

Reply:
But they should be string, right?

Reply:
Oh yeah, will fix.